CHANGES.md

```diff
@@ -16,6 +16,7 @@
 - Fixed artifact for skinned model when log depth is enabled. [#6447](https://github.com/CesiumGS/cesium/issues/6447)
 - Fixed a bug where certain rhumb arc polylines would lead to a crash. [#8787](https://github.com/CesiumGS/cesium/pull/8787)
 - Fixed handling of Label's backgroundColor and backgroundPadding option [#8949](https://github.com/CesiumGS/cesium/8949)
+- Fixed the issue that extension checking preprocessors (#ifdef GL_Extension) are not removed from glsl when upgrading to WebGL 2.0
```

Comment:
Maybe a higher level description like:

```suggestion
- Fixed log depth artifacts when running CesiumJS in a WebGL2 context. [#8969](https://github.com/CesiumGS/cesium/pull/8969)
```

Also make sure to include the PR number in the `CHANGES.md` entry like above: